comp/core/tagger/taggerimpl/tagstore/entity_tags.go

```diff
@@ -75,7 +93,26 @@ func (e *EntityTags) toEntity() types.Entity {
 	}
 }
 
-func (e *EntityTags) computeCache() {
+func (e *EntityTagsWithMultipleSources) getStandard() []string {
+	tags := []string{}
+	for _, t := range e.sourceTags {
+		tags = append(tags, t.standardTags...)
```

Comment:
shouldn't we make sure to remove duplication here?

Reply:
I didn't change this part. It's like this in the main branch:
https://github.com/DataDog/datadog-agent/blob/0634e14562761cee584c7ad81c794b473908c0b2/comp/core/tagger/taggerimpl/tagstore/entity_tags.go#L37